packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -1399,6 +1400,30 @@ export default class Meeting extends StatelessWebexPlugin {
     );
   }
 
+  /**
+   * post Event, provide the ability to send Call Analyzer for the request of third-party
+   * @public
+   * @memberof Meeting
+   * @returns {Promise}
+   */
+  // eslint-disable-next-line valid-jsdoc
+  public postEvent(eventName) {
+    Metrics.postEvent({
+      event: eventName,
+      meeting: this,
+    });
+  }
+
+  /**
+   * setCaCorrelationId, In the cross launch case, the correlationId sent by the page needs to be passed to prepare for the ucf CA report in the analyzer system.
+   * @public
+   * @memberof Meeting
+   */
+  // eslint-disable-next-line valid-jsdoc
+  public setCaCorrelationId(correlationId) {
```

Comment:
this method is not needed, we already have `setCorrelationId()` (see my other comment)